doc/source/faq.rst

```diff
@@ -114,9 +114,9 @@ Can I download climate model output in a "GIS-friendly" format?
 
 This question is related to the above question about Excel. Like spreadsheets, most (all?) GIS software packages are designed to display data in only two coordinate dimensions (i.e. a map). Suppose that you download daily data for a ten year period, how would your GIS software visualize the resulting 3600 layers? GIS software packages are not designed for this purpose. In general, you're going to need to do additional, needs-specific processing before you can create climate maps with your GIS software.
 
-In `version 2.1.0 <https://github.com/pacificclimate/pdp/blob/master/NEWS.rst#210>`_, we introduced the availability of the Arc/ASCII Grid format. This format only supports a single layer, so we deliver a Zip archive which contains one grid file per timestep. If downloading daily data, this is not recommended, but it *is* an option.
+Another option is that if your GIS software can speak WMS and you want to map individual time steps, please review our :ref:`power user HOWTO <power-user>`.
 
-One final option is that if your GIS software can speak WMS and you want to map individual time steps, please review our :ref:`power user HOWTO <power-user>`.
+In the past, we allowed support for the ArcInfo/ASCII Grid format, which consisted of a Zip archive containing one .asc file and one .prj (projection) file for each time step; however, this format is no longer supported as of this version of the data portal. This is because the recently added CMIP6 multivariate bias correction n-dimensional PDF transform (MBCn) data contains irregular lat/lon grids, which are not supported by this format. Additionally, we will soon be transitioning from our deployment of the `Pydap server <http://www.pydap.org/>`_ to the `THREDDS server <https://www.unidata.ucar.edu/software/tds/>`_, which does not support this format.
```

Comment:
Tighten wording:

> We no longer support the ArcInfo/ASCII Grid format[, which consisted of a Zip archive containing one .asc file and one .prj (projection) file for each time step]. The recently added CMIP6 multivariate bias correction n-dimensional PDF transform (MBCn) data contains irregular lat/lon grids, which are not supported by this format. Additionally, we will soon be transitioning from the `Pydap server <http://www.pydap.org/>`_ to the `THREDDS server <https://www.unidata.ucar.edu/software/tds/>`_, which does not support this format.

Consider removing the clause in the brackets [, which consisted of ... ]. Or else keep and remove the brackets.